.github/workflows/build.yml

```
@@ -19,4 +19,96 @@ jobs:
-      run: dotnet build TestStream.Runner/TestStream.Runner.csproj
\ No newline at end of file
+      run: dotnet build TestStream.Runner/TestStream.Runner.csproj
+    
+    - name: Trigger Azure DevOps Pipeline
+      env:
+        AZURE_DEVOPS_ORG: nanoframework
+        AZURE_DEVOPS_PROJECT: nanoFramework.IoT.TestStream
+        AZURE_DEVOPS_PIPELINE_ID: 111
+        AZURE_DEVOPS_PAT: ${{ secrets.AZURE_DEVOPS_PAT }}
+        AZURE_POOL_NAME: TestStream
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2140:warning:13:74: Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(shellcheck)